src/ImmutableArray.php

```diff
@@ -44,7 +44,7 @@ public function reverse($preserveKeys = false)
     /**
      * Pad array to the specified size with a given value
      *
-     * @param int $size Size of result array
+     * @param int   $size  Size of result array
```

Comment:
I think we should avoid this extra spaces.
Anyway, this changes should be in other PR not in this bug fix.